### PR TITLE
fix: validated leave allocation

### PR DESCRIPTION
**Pull request by Anurag810**
Screenshot 2020-02-17 at 11 48 00 AM
After Fix:
Screenshot 2020-02-17 at 12 02 01 PM

### CODE REVIEW

File: erpnext/hr/doctype/leave_encashment/leave_encashment.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ def get_leave_details_for_encashment(self):
 
 		allocation = self.get_leave_allocation()
 
+		if not allocation:
+			frappe.throw("No Leaves Allocated to Employee: {0} for Leave Type: {1}".format(self.employee, self.leave_type))
```
Comment (nabinhait) [inline erpnext/hr/doctype/leave_encashment/leave_encashment.py]: @Anurag810 Where is multi-lingual support for the message? This is bad :(

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/hr/doctype/leave_encashment/leave_encashment.py (hunk 1 above)
Comment (Anurag810) [reply erpnext/hr/doctype/leave_encashment/leave_encashment.py]: Done

File: erpnext/hr/doctype/leave_encashment/leave_encashment.py [hunk 2]
```diff
@@ -64,6 +64,9 @@ def get_leave_details_for_encashment(self):
 
 		allocation = self.get_leave_allocation()
 
+		if not allocation:
+			frappe.throw(_("No Leaves Allocated to Employee: {0} for Leave Type: {1}".format(self.employee, self.leave_type)))
```
Comment (nabinhait) [inline erpnext/hr/doctype/leave_encashment/leave_encashment.py]:
```suggestion
			frappe.throw(_("No Leaves Allocated to Employee: {0} for Leave Type: {1}").format(self.employee, self.leave_type))
```

**Review by nabinhait — CHANGES_REQUESTED** [review]

**Comment by Anurag810** [comment]
@nabinhait please review and merge

**Review by nabinhait — APPROVED** [review]

**Pull request merged**